clouddrift/datasets.py

```diff
@@ -157,6 +157,49 @@ def gdp6h(decode_times: bool = True) -> xr.Dataset:
     return ds
 
 
+def gdpraw(
+    tmp_path: str = adapters.rawfiles._TMP_PATH,
+    max: int | None = None,
+    decode_times: bool = True,
+) -> xr.Dataset:
+    """Returns the NOAA Global Drifter Program (GDP) raw dataset as a ragged array
```

Comment:
Let's replace here and below "raw" by "un-interpolated"

Reply:
what about the funcitons' name? Looks like we have `def gdpraw(` for example.

Reply:
What about `gdp_source` ?  and replace raw dataset with source (uninterpolated) dataset.

Reply:
do we want `gdp_source` , `gdp_uninterpolated`. We could also use something like `gdp_preprocess`